eZ/Bundle/EzPublishCoreBundle/Command/CopySubtreeCommand.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Bundle\EzPublishCoreBundle\Command;
+
+use eZ\Publish\API\Repository\ContentTypeService;
+use eZ\Publish\API\Repository\LocationService;
+use eZ\Publish\API\Repository\PermissionResolver;
+use eZ\Publish\API\Repository\UserService;
+use eZ\Publish\Core\Base\Exceptions\InvalidArgumentException;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class CopySubtreeCommand extends Command
+{
+    /**
+     * @var \Symfony\Component\Console\Input\InputInterface
+     */
+    private $input;
+
+    /**
+     * @var \Symfony\Component\Console\Output\OutputInterface
+     */
+    private $output;
+
+    /**
+     * @var \eZ\Publish\API\Repository\LocationService
+     */
+    private $locationService;
+
+    /**
+     * @var \eZ\Publish\API\Repository\PermissionResolver
+     */
+    private $permissionResolver;
+
+    /**
+     * @var \eZ\Publish\API\Repository\UserService
+     */
+    private $userService;
+
+    /**
+     * @var \eZ\Publish\API\Repository\ContentTypeService
+     */
+    private $contentTypeService;
+
+    /**
+     * @param \eZ\Publish\API\Repository\LocationService $locationService
+     * @param \eZ\Publish\API\Repository\PermissionResolver $permissionResolver
+     * @param \eZ\Publish\API\Repository\UserService $userService
+     * @param \eZ\Publish\API\Repository\ContentTypeService $contentTypeService
+     */
+    public function __construct(
+        LocationService $locationService,
+        PermissionResolver $permissionResolver,
+        UserService $userService,
+        ContentTypeService $contentTypeService
+    ) {
+        parent::__construct();
+        $this->locationService = $locationService;
+        $this->permissionResolver = $permissionResolver;
+        $this->userService = $userService;
+        $this->contentTypeService = $contentTypeService;
+    }
+
+    protected function configure()
+    {
+        $this
+            ->setName('ezplatform:copy-subtree')
+            ->addArgument(
+                'source-location-id',
+                InputArgument::REQUIRED,
+                'Id of subtree root location'
+            )
+            ->addArgument(
+                'target-location-id',
+                InputArgument::REQUIRED,
+                'Id of target location'
+            )
+            ->addOption(
+                'user',
+                'u',
+                InputOption::VALUE_OPTIONAL,
+                'eZ Platform username (with Role containing at least Content policies: read, versionread, edit, remove, versionremove)',
```

Comment:
Have you checked described here list of policies, do they actually reflect the need of this command?